### PR TITLE
CAMEL-13617: Improve testability of google-sheets component

### PR DESCRIPTION
**Pull request by christophd**
also incorporate latest component enhancements to Camel 3.x

### CODE REVIEW

File: components/camel-google-sheets/pom.xml [hunk 1]
```diff
@@ -39,13 +39,59 @@
         <componentPackage>org.apache.camel.component.google.sheets</componentPackage>
         <outPackage>org.apache.camel.component.google.sheets.internal</outPackage>
         <camel.osgi.private.pkg>org.apache.camel.component.google.sheets.internal</camel.osgi.private.pkg>
+
+        <citrus.version>2.8.0</citrus.version>
```
Comment (oscerd) [inline components/camel-google-sheets/pom.xml]: Here too: maybe we can move the placeholders in parent/POM

Comment (christophd) [reply components/camel-google-sheets/pom.xml]: Done!

**Comment by oscerd** [comment]
Merged, I just changed the codestyle a bit, you can check this through the maven profile sourcecheck

**Comment by oscerd** [comment]
Thanks!

**Pull request closed without merging**